src/trace.ml

```diff
@@ -573,11 +610,14 @@ module Make_commands (Backend : Backend_intf.S) = struct
          magic-trace run -multi-thread ./program -- arg1 arg2\n\n\
          # Run a process, tracing its entire execution (only practical for short-lived \
          processes)\n\
-         magic-trace run -full-execution ./program\n")
+         magic-trace run -full-execution ./program\n\
+         # Run a process that generates a large trace file, magic-trace run ./program \
+         -use-trace-processor-shell\n")
```

Comment:
We should leave this out of the documentation for now, or maybe make the `-use-trace-processor-shell` part of it conditional on the environment variable being present. Definitely don't want to give help text for a flag that doesn't exist.